maria_sirley/js/maria_hello_word_js.index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mi primer Script</title>
+</head>
+
+<body>
+    <p>Este es mi primer mensaje con JavaScript en el curso Full Stack JS web Developer</p>
+</body>
+
+<script src="maria_sirley\js\alert.js"></script>
+
```

Comment:
Esto debe estar antes de cerrar la etiqueta body

```suggestion
  <script src="maria_sirley\js\alert.js"></script>
</body>
```